packages/app/src/ui/components/modal-dialog/modal-dialog.component.tsx

```diff
@@ -86,7 +87,7 @@ export const ModalDialog: FunctionComponent<ModalDialogProps> = withTheme(
         isOpen={open}
         // AUTO DISABLED - fix me if you like :)
         // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-        style={getModalDialogStyle({ zIndex, size, theme })}
+        style={getModalDialogStyle({ zIndex: layer, size, theme })}
```

Comment:
so, basically, there will be still zIndex underneath every modal, but from now on we call it layer? :D

Reply:
pretty much, yeah. I remember that idea to call it layer or something similar originated during my conversation with @lkowalczyk87 but ofc this is not mandatory. I can change the name to zIndex or smth, if majority will agree to diffirent a name.